Boards: File attachments (#6141)
mattermost/docs: source/boards/work-with-cards.rst

Once you have card properties defined, you have full control over which properties are shown on the board as a preview without having to open the card. Select **Properties** at the top of the board, then enable all properties you want to see at a glance, and hide all properties you don’t want to see.

Attach files
------------

> From Mattermost v7.7 you can attach files to your cards, which viewers can view and download. There are no limitations to the file types that you can upload.

Reviewer: ```suggestion
From Mattermost v7.7 you can attach files to your cards, which other board members can download. There are no limitations to the file types that you can upload.
```

> To upload a file to a card, select **Attach** in the top-right corner of the card. Then select the file or files you'd like to upload. You can upload up to 10 files at a time. When your file has been uploaded, you can find it in the **Attachments** section of the card. Select the **+** sign to add additional files to your card.

Reviewer: ```suggestion
To upload a file to a card, select **Attach** in the top-right corner of the card. Then select the file you'd like to upload. When your file has been uploaded, you can find it in the **Attachments** section of the card. Select the **+** sign to add additional files to your card.
```
